elm/lib/dependabot/elm/file_parser.rb

```diff
@@ -167,12 +186,17 @@ def version_for(version_requirement)
         req.requirements.first.last
       end
 
+      sig { returns(T::Hash[String, T.any(String, T::Hash[String, T.any(String, T::Hash[String, String])])]) }
```

Comment:
I would monitor this after deploying. It's too precise and it wouldn't surprise me if it starts errorong.